src/main/java/org/immregistries/mismo/match/PatientMatchResult.java

```diff
@@ -0,0 +1,42 @@
+package org.immregistries.mismo.match;
+
+public class PatientMatchResult {
+
+  private PatientMatchDetermination determination = null;
+  private String signatureLevel0 = "";
+  private String signatureLevel1 = "";
+  private String signatureLevel2 = "";
```

Comment:
Another thought would be to provide a List of signatures, in case we ever decide to provide more or fewer levels? Though I haven't dived into the level code too deeply to understand it.

Reply:
Good idea. I'm going to create a Signature object that will just hold a string. But could hold more. I think I'll return it as a list of signature objects. The order will be consistent, but they will also be identified in the object as to what type of signature it is.